README.md

```diff
@@ -56,18 +56,24 @@ Files in .gitignore (auto-generated)
 For functions not yet implemented:
 
 ```js
-function function_name()) {
+function functionName()) {
     throw new Error(`TODO: Not implemented`);
 }
 ```
 
-And for functions that will be not implemented:
+And for functions or classes that will not be implemented:
 
 ```js
 /**
- * @deprecated The method should not be used
+ * @deprecated Not supported by the driver. Usage will throw an error.
  */
-function function_name() {
-  throw new ReferenceError(`This function is not supported by our driver`);
+function functionName() {
+  throw new ReferenceError(`The function (functionName) is not supported by our driver`);
 }
+
+/**
+ * @deprecated Not supported by the driver. Usage will throw an error.
+ */
+function ClassName() {
+  throw new ReferenceError(`The class (ClassName) is not supported by our driver`);
```

Comment:
Are classes in JS functions, too??

Reply:
In the older versions yes, now changed to new syntax.